### PR TITLE
Mission component styling

### PR DESCRIPTION
**Pull request by udeaghad**
In this pull request,

- I have styled the mission component and the profile.

Kindly review and approve

### CODE REVIEW

File: src/styles/profile.css [hunk 1]
```diff
@@ -6,18 +6,38 @@
...
+}
+
+li {
+  list-style: none;
+}
+
+.mission-item {
+  border-bottom: 1px solid gray;
+  padding: 4%;
+  font-size: 20px;
```
Comment (Zelalem1222) [inline src/styles/profile.css:42]: It would be great if you decrease the top and bottom padding. 
 padding : 2% 4%;

File: src/styles/profile.css [hunk 2]
```diff
@@ -6,18 +6,38 @@
 
 .reserved-rocket {
   border: 1px solid lightgrey;
-  padding: 3% 3%;
   margin: 0;
+  padding: 4%;
+  font-size: 20px;
```
Comment (Zelalem1222) [inline src/styles/profile.css:11]: Also decrease the padding.
padding : 2% 4%;

**Review by Zelalem1222 — APPROVED** [review]
## Great work @udeaghad .

check the comments below and kindly fix them.

File: src/styles/profile.css (hunk 1 above)
Comment (udeaghad) [reply src/styles/profile.css:42]: Done

File: src/styles/profile.css (hunk 2 above)
Comment (udeaghad) [reply src/styles/profile.css:11]: done

**Pull request merged**